### ISSUE TITLE
Allow passing of class names

### ISSUE THREAD

**Issue by erwstout** [factor1/react-card-flipper#6]
Right now, if you add a `className` to `ReactCardFlipper` it does not accept it. Should be adjusted to allow passing of class names.

```js
<ReactCardFlipper width="100%" height="300px" levitate={true} className="your-class-name">
```

**Comment by erwstout** [comment]
PR #7 should do the trick, will test and merge in soon.

**Comment by erwstout** [comment]
Merged and released on `0.3.0`
